code/common/datastripe.q

```diff
@@ -1,3 +1,26 @@
+// stripe data across rdbs
+
 \d .ds
 
 segmentid: "J"$.proc.params[`segid]		// segmentid variable defined by applying key to dictionary of input values
+
+\d .
+
+.rdb.datastripeendofperiod:{[currp;nextp;data]
+    .lg.o[`reload;"reload command has been called remotely"];
+    
+    // remove periods of data from tables
+    t:tables[`.] except .rdb.ignorelist;
+    lasttime:currp-.ds.periodstokeep*(nextp-currp);
+    tabs:{![x;enlist (<;y;z);0b;0#`]}'[t;`time;lasttime];
```

Comment:
I think this should be a separate function as opposed to a lambda.  It seems useful and could potentially be reused?